tests/test_save.py

```diff
@@ -107,7 +108,7 @@ def example_save(self, id):
         if os.path.exists(get_save_dir()):
             shutil.rmtree(get_save_dir())
 
-        #copy tests/testSaves/save<id> to saves
+        # tests/testSaves/save<id> to saves
```

Comment:
Why are we removing the word copy here?

Reply:
Oh sorry, that looks like a mistake.  Probably accidentally selected the word while trying to add a space.

Reply:
Fixed it now.